packages/cli/src/ui/hooks/atCommandProcessor.test.ts

```diff
@@ -31,7 +33,7 @@ describe('handleAtCommand', () => {
   async function createTestFile(fullPath: string, fileContents: string) {
     await fsPromises.mkdir(path.dirname(fullPath), { recursive: true });
     await fsPromises.writeFile(fullPath, fileContents);
-    return path.resolve(testRootDir, fullPath);
+    return escapePath(normalizePath(path.resolve(testRootDir, fullPath)));
```

Comment:
why does createTestFile return an escaped path? seems like that could hide some bugs.